google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/model/QueryJsonProtocol.scala

```diff
@@ -31,14 +31,18 @@ import scala.concurrent.duration.FiniteDuration
  * @param dryRun if set to `true` BigQuery doesn't run the job and instead returns statistics about the job such as how many bytes would be processed
  * @param useLegacySql specifies whether to use BigQuery's legacy SQL dialect for this query
  * @param requestId a unique user provided identifier to ensure idempotent behavior for queries
+ * @param location the geographic location where the job should run
+ * @param maximumBytesBilled limits the number of bytes billed for this query
  */
 final case class QueryRequest private (query: String,
                                        maxResults: Option[Int],
                                        defaultDataset: Option[DatasetReference],
                                        timeout: Option[FiniteDuration],
                                        dryRun: Option[Boolean],
                                        useLegacySql: Option[Boolean],
-                                       requestId: Option[String]) {
+                                       requestId: Option[String],
+                                       location: Option[String],
+                                       maximumBytesBilled: Option[Long]) {
```

Comment:
Just a tiny tiny nitpick, could we keep the order consistent with the [BigQuery reference](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#QueryRequest)? Here and in `apply`/`create`. Not a huge deal :)

Reply:
Thanks! No problems, I've changed the order in the private constructor.